app/views/layouts/curation_concerns/1_column.html.erb

```diff
@@ -26,6 +26,7 @@
       <%= yield %>
     </div>
   </div><!-- /.row -->
+<%= render partial: '/permalink' %>
```

Comment:
This looks good on works but has awkward layout on user-show page.

![screen shot 2017-01-20 at 10 34 03 am](https://cloud.githubusercontent.com/assets/1069588/22155126/572237a4-defc-11e6-838e-01d9293d764f.png)